Group explorer

brodeur opened the pull request:
New group explorer, who dis? 

Improvements include styling, a new featured groups section.

Also included is an upgrade to our public views of posts, groups & map

thomasgwatson commented on apps/web/src/routes/GroupSettings/DeleteSettingsTab/DeleteSettingsTab.js:
```diff
@@ -1,41 +1,46 @@
 
   const handleDeleteGroup = useCallback(() => {
-    if (window.confirm(t('Are you sure you want to delete the group {{name}}?', { name }))) {
+    if (window.confirm(t('Are you sure you want to delete the group {{name}}? This will permanently delete the group and all of its content. This cannot be undone.', { name }))) {
```
So, this changes the 'key' to a translation, and then, since that new key doesn't exist, it will miss the key lookup and just display the key. So now it will no longer translate to spanish.

To change the copy, you need to make sure there is an entry in the locale files that matches the key. And you can change the values of those entries to the desired copy

brodeur commented on apps/web/src/routes/GroupExplorer/FeaturedGroups/FeaturedGroups.js:
```diff
@@ -0,0 +1,115 @@
+      <div className='flex items-center justify-between'>
+        <h2 className='mt-0 font-bold ml-4 text-sm'>{t('Featured Groups')}</h2>
+        <div className='relative group mr-4 w-6 h-6 flex items-center justify-center' data-tooltip-id='featured-groups-info'>
+          <Link to={groupDetailUrl('building-hylo', routeParams)}>
```
Not sure how to do this correctly. It's for a tooltip that says "To recommend a group to be featured, join building Hylo"

brodeur commented on apps/web/src/routes/GroupExplorer/GroupExplorer.js:
```diff
@@ -1,33 +1,36 @@
   }, [])
 
+  // Building Hylo, Hylo Alliance, Hylo Community Organizers, CTA, PHA ...?
+  const featuredGroupIds = ['20866', '36177', '20966', '20828', '39912']
```
Moved this to an env variable, lets discuss which groups to include today

brodeur commented on apps/web/src/contexts/ViewHeaderContext/ViewHeaderProvider.jsx:
```diff
@@ -7,8 +7,12 @@ export const ViewHeaderProvider = ({ children }) => {
-    title: '',
-    search: false
+    title: {
+      mobile: '',
```
I fixed viewheader so that it accepts both types of title

brodeur commented:
I've resolved the translation issues and addressed the above comments. I think this is ready after our conversations about which groups to include in the recommended groups area

@tibetsprague @thomasgwatson
